Makefile

```diff
@@ -100,3 +100,7 @@ verify-bindata-v4.1.0: verify-apirequestcounts-crd
 .PHONY: verify-apirequestcounts-crd
 verify-apirequestcounts-crd:
 	diff -Naup $(APIREQUESTCOUNT_CRD_SOURCE) $(APIREQUESTCOUNT_CRD_TARGET)
+
+# This should be used by CI only
+build_with_dev_cert_rotation: GO_LD_EXTRAFLAGS :=-X $(GO_PACKAGE)/pkg/operator/certrotationcontroller.certRotationAccelerator="60"
```

Comment:
having our nightlies rotate quickly means that every test and developer cluster we produce until we ship  rotates quickly.  Changing this means that we will no longer get any testing (CI doesn't run long enough to trigger a rotation).